daft/expressions/expressions.py

```diff
@@ -164,6 +164,38 @@ def col(name: str) -> Expression:
     return Expression._from_pyexpr(_col(name))
 
 
+def list_(*items: Expression | str):
```

Comment:
> If a function argument’s name clashes with a reserved keyword, it is generally better to append a single trailing underscore rather than use an abbreviation or spelling corruption. Thus class_ is better than clss. (Perhaps better is to avoid such clashes by using a synonym.) https://peps.python.org/pep-0008/#function-and-variable-names